README.md

```diff
@@ -40,8 +40,8 @@ Range, Readings
 
 Establish quality parameters: 
 
-- What is the maximum complexity (CCN) per function? _enter CCN and create corresponding yml in the `.github/workflows` folder
-- How many lines of duplicate code will you tolerate? _enter the number of lines and create corresponding yml in the `.github/workflows` folder
+- What is the maximum complexity (CCN) per function? 3 CCN and create corresponding yml in the `.github/workflows` folder
```

Comment:
You have kept the CCN limit as 4 in the yml, not 3

Reply:
My goal is CCN limit as 3. Currently, I have working towards this goal to achieve.

Reply:
Done. Now Goal achieved. CCN value is 3 in both readme & yml.